convert2rhel/actions/pre_ponr_changes/kernel_modules.py

```diff
@@ -215,33 +215,49 @@ def run(self):
                     " We will continue the conversion with the following kernel modules unavailable in RHEL:\n"
                     "{kmods}\n".format(kmods="\n".join(unsupported_kmods))
                 )
+                self.add_message(
+                    level="WARNING",
+                    id="ALLOW_UNAVAILABLE_KERNEL_MODULES",
+                    title="Skipping the ensure kernel modules compatibility check",
+                    description="Detected 'CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS' environment variable.",
+                    diagnosis="We will continue the conversion with the following kernel modules unavailable in RHEL:\n"
+                    "{kmods}\n".format(kmods="\n".join(unsupported_kmods)),
+                )
                 return
 
-            # If there is any unsupported kmods found, set the result to error
+            # If there is any unsupported kmods found, set the result to overridable
             if unsupported_kmods:
                 self.set_result(
-                    level="ERROR",
+                    level="OVERRIDABLE",
                     id="UNSUPPORTED_KERNEL_MODULES",
-                    message=(
-                        "The following loaded kernel modules are not available in RHEL:\n{0}\n"
-                        "Ensure you have updated the kernel to the latest available version and rebooted the system.\nIf this "
-                        "message persists, you can prevent the modules from loading by following {1} and rerun convert2rhel.\n"
-                        "Keeping them loaded could cause the system to malfunction after the conversion as they might not work "
-                        "properly with the RHEL kernel.\n"
-                        "To circumvent this check and accept the risk you can set environment variable "
-                        "'CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS=1'.".format(
-                            "\n".join(unsupported_kmods), LINK_PREVENT_KMODS_FROM_LOADING
-                        )
+                    title="Unsupported kernel modules found",
+                    diagnosis="The following loaded kernel modules are not available in RHEL:\n{0}\n".format(
+                        "\n".join(unsupported_kmods)
                     ),
+                    remediation="Ensure you have updated the kernel to the latest available version and rebooted the system.\nIf this "
+                    "message persists, you can prevent the modules from loading by following {1} and rerun convert2rhel.\n"
```

Comment:
I think that CodeQL is failing this message because of the `{1}` in the message, as it should be `{0}` instead, since we only have one argument... 

Maybe it is because of that, as this message doesn't seem to contain another argument to format...